libraries/grpc-sdk/src/utilities/normalizeParams.ts

```diff
@@ -0,0 +1,10 @@
+export function normalizeParams(givenParams: any[], fnParamKeys: string[]) {
+  if (givenParams.length === 1 && typeof givenParams[0] === 'object') {
```

Comment:
What if the array-variant expects an object arg at pos[0]?
This would only pass in the first arg, assuming object formatting.

Reply:
I changed the logic, now only optional params can be set as an object